cmd/did-rest/startcmd/test/config.json

```diff
@@ -4,6 +4,10 @@
       "pattern": "^(did:trustbloc:testnet.trustbloc.local:.+)$",
       "url": "http://trustbloc.did.method.example.com:8060/resolveDID?did=$1"
     },
+    {
+      "pattern": "^(did:key:.+)$",
+      "vdri": true
```

Comment:
I wonder if this is the most descriptive configuration property?

thoughts? @fqutishat @sandrask @bstasyszyn @aholovko

Reply:
It is descriptive from developer point since we use VDRI :) How about embedded or local? Not sure those names are any better.

Reply:
I suppose that the "full" configuration will be something like this:
```json
{
  "pattern": "^(did:btcr:.+)$",
  "image": "universalresolver/driver-did-btcr",
  "imageProperties": "true",
  "tag": "latest",
  "testIdentifiers": ["did:btcr:xz35-jznz-q9yu-ply"]
}
```

Perhaps `"local":true` looks not so bad among other properties. It denotes that it's not the docker image but the local implementation.

Reply:
we should not expose  "vdri": true

Reply:
@fqutishat we all agree on that - we are just trying to come up with meaningful name - I am ok with local @troyronda?

Reply:
sure - local, embedded, system would be possibilities.

Reply:
no field is needed just ```"pattern": "^(did:key:.+)$"```

Reply:
I suppose that's true. If url and image are not present, I guess it's implied that it is a built-in handler.

Reply:
I'm looking at this code for the first time.

To me it looks like `TransformResult` should be a resolver.

Reply:
If we remove all keys and just leave `pattern` in this config object, then are we left with "rules" or just a whitelist?